pkg/smokescreen/config.go

```diff
@@ -82,6 +82,9 @@ type Config struct {
 	// Customer handler to allow clients to modify reject responses
 	RejectResponseHandler func(*http.Response)
 
+	// Customer handler to allow clients to modify accept responses
```

Comment:
Reading this I'm wondering if the comment was supposed to say "custom" 🤔

Reply:
Probably!